deploy/AzureAppServices/Datadog.Trace.AzureAppServices.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Trace.AzureAppServices</id>
+    <version>1.10.39-prerelease</version>
+    <title>.NET Datadog APM [Beta]</title>
+    <authors>Datadog</authors>
+    <icon>icon.png</icon>
+    <license type="expression">Apache-2.0</license>
```

Comment:
nit: Isn't the MIT license the usual choice for this type of thing?

Reply:
Our repo is Apache-2.0 so it makes sense that this would also be Apache-2.0

Reply:
ok. Makes sense. I was thinking of all site extensions, which may be different, and have numerous different licenses.